packages/manager/src/components/DatePicker/DateTimeRangePicker.tsx

```diff
@@ -73,72 +76,148 @@ export const DateTimeRangePicker = ({
+        newEndDateTime = null;
+        break;
+      default:
+        return;
+    }
+
+    setStartDateTime(newStartDateTime);
+    setEndDateTime(newEndDateTime);
+    setPresetValue(value);
+
+    if (onChange) {
+      onChange(newStartDateTime, newEndDateTime, startTimeZone, value);
+    }
+
+    setShowPresets(value !== 'custom_range');
   };
 
   const handleStartDateTimeChange = (newStart: DateTime | null) => {
     setStartDateTime(newStart);
     validateDates(newStart, endDateTime, 'start');
 
-    onChange(newStart, endDateTime, startTimeZone);
+    if (onChange) {
+      onChange(newStart, endDateTime, startTimeZone, 'custom_range');
+    }
   };
 
   const handleEndDateTimeChange = (newEnd: DateTime | null) => {
     setEndDateTime(newEnd);
     validateDates(startDateTime, newEnd, 'end');
 
-    onChange(startDateTime, newEnd, startTimeZone);
-  };
-
-  const handleStartTimeZoneChange = (newTimeZone: null | string) => {
-    setStartTimeZone(newTimeZone);
-
-    onChange(startDateTime, endDateTime, newTimeZone);
+    if (onChange) {
+      onChange(startDateTime, newEnd, startTimeZone, 'custom_range');
+    }
   };
 
   return (
-    <Box display="flex" gap={2} sx={sx}>
-      {/* Start DateTime Picker */}
-      <DateTimePicker
-        timeZoneSelectProps={{
-          label: 'Start TimeZone',
-          onChange: handleStartTimeZoneChange,
-          value: startTimeZone,
-        }}
-        errorText={startDateError ?? startDateErrorMessage}
-        format={format}
-        label={startLabel}
-        onChange={handleStartDateTimeChange}
-        showTimeZone={showStartTimeZone}
-        timeSelectProps={{ label: 'Start Time' }}
-        value={startDateTime}
-      />
-
-      {/* End DateTime Picker */}
-      <DateTimePicker
-        timeZoneSelectProps={{
-          value: startTimeZone, // Automatically reflect the start timezone
-        }}
-        errorText={endDateError ?? endDateErrorMessage}
-        format={format}
-        label={endLabel}
-        onChange={handleEndDateTimeChange}
-        showTimeZone={showEndTimeZone}
-        timeSelectProps={{ label: 'End Time' }}
-        value={endDateTime}
-      />
+    <Box display="flex" flexDirection="column" gap={2} sx={sx}>
+      {showPresets ? (
+        <Autocomplete
+          onChange={(_, selection) => {
+            if (selection) {
+              handlePresetSelection(selection.value);
+            }
+          }}
+          options={[
+            { label: 'Last 24 Hours', value: '24hours' },
+            { label: 'Last 7 Days', value: '7days' },
+            { label: 'Last 30 Days', value: '30days' },
+            { label: 'This Month', value: 'this_month' },
+            { label: 'Last Month', value: 'last_month' },
+            { label: 'Custom', value: 'custom_range' },
+          ]}
+          value={
+            presetValue
+              ? { label: presetValue.replace('_', ' '), value: presetValue }
+              : null
+          }
```

Comment:
```suggestion
          }
         isOptionEqualToValue={(option, value) => option.value === value.value}
```

This will remove the warning as the object we are passing to the value prop has value of `label` key same as value of `value` key but it is actually different in options list